recipes/optking/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "optking" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/psi-rking/optking/archive/{{ version }}.tar.gz
+  sha256: 57c5b15fc160a6ed7cf260f669f332d97fd74ff5f0eb6cad9cf27d65d049d24a 
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```

Should be the same as `run`.
```suggestion
    - python >=3.7
```